.github/workflows/python-ci.yml

```diff
@@ -31,10 +31,15 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install yapf
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
is there a chance this file will not be present here? Actually, I think the CI **should** fail if, for any reason, `requirements.txt` is not present

Reply:
that's a good point, changed thanks